traits/ctraits.c

```diff
@@ -185,94 +187,93 @@ static int call_notifiers ( PyListObject *, PyListObject *,
    validation. */
 #define MAXIMUM_DEFAULT_VALUE_TYPE 9
 
-
 /*-----------------------------------------------------------------------------
 |  'CTrait' instance definition:
 +----------------------------------------------------------------------------*/
 
 typedef struct _trait_object a_trait_object;
-typedef PyObject * (*trait_getattr)( a_trait_object *, has_traits_object *,
-                                     PyObject * );
-typedef int (*trait_setattr)( a_trait_object *, a_trait_object *,
-                              has_traits_object *, PyObject *, PyObject * );
-typedef int (*trait_post_setattr)( a_trait_object *, has_traits_object *,
-                                   PyObject *, PyObject * );
-typedef PyObject * (*trait_validate)( a_trait_object *, has_traits_object *,
-                              PyObject *, PyObject * );
-typedef PyObject * (*delegate_attr_name_func)( a_trait_object *,
-                                             has_traits_object *, PyObject * );
+typedef PyObject *(*trait_getattr)(
+    a_trait_object *, has_traits_object *, PyObject *);
+typedef int (*trait_setattr)(
+    a_trait_object *, a_trait_object *, has_traits_object *, PyObject *,
+    PyObject *);
+typedef int (*trait_post_setattr)(
+    a_trait_object *, has_traits_object *, PyObject *, PyObject *);
+typedef PyObject *(*trait_validate)(
+    a_trait_object *, has_traits_object *, PyObject *, PyObject *);
+typedef PyObject *(*delegate_attr_name_func)(
+    a_trait_object *, has_traits_object *, PyObject *);
 
 typedef struct _trait_object {
     PyObject_HEAD                    /* Standard Python object header */
-    unsigned int       flags;        /* Flag bits */
-    trait_getattr      getattr;      /* Get trait value handler */
-    trait_setattr      setattr;      /* Set trait value handler */
+        unsigned int flags;          /* Flag bits */
```

Comment:
manual adjustment needed here, too

Reply:
No manual adjustment needed: clang handles this fine with a suitable StatementMacro declaration in the `.clang-format` file.